src/index.js

```diff
@@ -25,7 +25,7 @@ function jsxToString(component, options) {
       (key) => {
         if (key === 'children' ||
           isDefaultProp(component.type.defaultProps, key, component.props[key]) ||
-          opts.ignoreProps.includes(key)) {
+            opts.ignoreProps.indexOf(key) > -1) {
```

Comment:
I believe there is a spacing issue in this line. Would you mind to remove the two extra characters ?